Automattic/woocommerce-payments
# Reporting: Payment activity widget wrapper - Remove unwanted CSS classes

Fixes #8389

#### Changes proposed in this Pull Request

This is a refactor of a PR - #8412 ,  I merged earlier. It removes unwanted CSS from the Card and its sub-components. 

#### Testing instructions
* Make sure you have the required feature flag enabled 
  *  Via CLI, add the feature flag to the client site using the following command:
`wp option update _wcpay_feature_payment_overview_widget 1`

* The Payment activity widget should appear similar to the screenshot:

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/0ae0abc9-ff64-44fa-b5d3-e59358e4edaa)

#### Dev notes

Kindly review the relevant commit here - https://github.com/Automattic/woocommerce-payments/pull/8507/commits/41be9c16b845c26a9f67a5156d441b59e6726a1b . This contains all the changes I made for this PR. 

All other commits appear because I had not merged changes from develop prior. Sorry about the clutter. 


-------------------


**Post merge**
